src/io/opendss.jl

```diff
@@ -1,6 +1,14 @@
 # OpenDSS parser
 
 
+"parses 'source_id' into Dict containing the 'type', 'name' and active 'phases' of the OpenDSS component"
+function parse_source_id(source_id::String)
```

Comment:
probably should be called something like `parse_opendss_source_id`, for cases when `source_id` comes from a non opendss file.

Reply:
Also, if you choose to break up the component name and phase info into two separate fields then this function could take as input a component Dict rather than a string.

In that case, probably the name should also be revised.